guides/micronaut-graphql-todo/metadata.json

```diff
@@ -0,0 +1,15 @@
+{
+  "title": "Creating a ToDo application with Micronaut GraphQL",
+  "intro": "Build a TODO application with Micronaut GraphQL.",
+  "authors": ["Sergio del Amo", "Tim Yates"],  
+  "category": "GraphQL",
+  "tags": ["flyway", "micronaut-data", "jdbc", "graphql"],
+  "publicationDate": "2022-03-10",
+  "languages": ["java"],
+  "apps": [
+    {
+      "name": "default",
+      "features": ["graphql", "data-jdbc", "flyway", "jdbc-hikari" ,"postgres", "testcontainers", "graalvm"]
```

Comment:
no need to specify jdbc-hikari, it's included by the data-jdbc feature

Reply:
Thanks 👍  Gone